### PR TITLE
Kodi: Remove "disable addon dialog at startup" for LibreELEC

### PR DESCRIPTION
**Pull request by mglae**
The feature is not only not needed for LE but as well does enable the DVB driver addons.

Fix for #5397

Tested on Generic.

### CODE REVIEW

**Comment by CvH** [comment]
asked at Kodi PR for clarification https://github.com/xbmc/xbmc/pull/19091, otherwise we need to carry that patch forever :/

**Comment by chewitt** [comment]
It should probably be made configurable via `appliance.xml` or similar.

**Comment by howie-f** [comment]
is it problematic to carry the patch with you? sorry for the lack of experience, i think it‘s the correct way to disable it for LE

**Comment by CvH** [comment]
> is it problematic to carry the patch with you?

more patches = more problems :)

Overall its not to problematic to carry that patch, but if some solution went upstream nobody would complain either.
Likely this is a *ELEC specific problem and nothing that linux distros has to deal in general.

**Review by CvH — APPROVED** [review]

**Comment by Portisch** [comment]
Just by interests: is this a "work around" for the pop-up on Kodi start because the addon is disabled after install? :thinking: 

I might be wrong but I think this is caused by a bug in:
https://github.com/mglae/LibreELEC.tv/blob/267903e4a79e2c3f0d579e66edb080ee50d9d848/packages/mediacenter/kodi/patches/kodi-100.09-use-a-wrapper-to-setup-service-addons.patch#L45-L47

We fixed it this way:
https://github.com/CoreELEC/xbmc/commit/609fd49a1351e56af1e537d1fb667b0170fbd7eb#diff-62fdfe3ec618922fd79835e9be13b8a8584acc29f9f7072137c2b94fd44634b0R445-R448
There got an additional `LE_ADDON_ENABLED` added as otherwise only on update the addon get enabled.
You easily can check it when install the addon twice. Then you will see it get enabled and no pop-up shows up.

With this additional `LE_ADDON_ENABLED` the addon is already enabled after first install and no pop-up will show up at all. So there is no need to remove the pop-up.

But again, I am maybe wrong and talking about something different...

**Comment by CvH** [comment]
at a clean install the bundled addons (that should stay disabled by default) ask at first start of kodi "do you like to enable addon ..." for every bundled addon (also at 18->19 update)
this is something new, didn't happens at K18

iirc @SupervisedThinking told at some point that all his bundled addons are also asked if you like to enable it even if enabled ?? so maybe some additional bug here

so basically that only touches bundled addons at the image - nothing that came from the repo

**Comment by Portisch** [comment]
Yes, we saw the same. Especially update from v18 to v19. The message was gone buy fixing the service wrapper.

**Comment by CvH** [comment]
@howie-f any opinion what is the better solution ?
https://github.com/CoreELEC/xbmc/commit/609fd49a1351e56af1e537d1fb667b0170fbd7eb#diff-62fdfe3ec618922fd79835e9be13b8a8584acc29f9f7072137c2b94fd44634b0R445-R448
or adding the patch https://github.com/LibreELEC/LibreELEC.tv/pull/5403/commits/267903e4a79e2c3f0d579e66edb080ee50d9d848 ?

**Comment by Portisch** [comment]
Maybe instead call enable the same can be done by using `LE_ADDON_DISABLED`. vpeter4 remembered that LE prefer to disable addon on install what does not make much sense in our opinion. But this is something different...

https://github.com/LibreELEC/LibreELEC.tv/pull/5251

But maybe using `LE_ADDON_DISABLED` instead `LE_ADDON_ENABLED` should do the trick.

**Comment by howie-f** [comment]
the flag is an option and i was thinking about something similar yesterday. but honestly i would stick with the patch. the purpose of the original change was: linux folks may install their binary add-ons via `apt install pvr.hts...`, so they should be asked to enable and configure it on next kodi start. but this behaviour caused issues on other platforms like `UWP` where multiple pvr addons are shipped (and have to stay disabled). same as the LE situation now i guess. as there is no package manager with LE i would just patch it out and be good :)

was also thinking about a cmake flag approach similar to `PLATFORM=rbpi` in the past, lets call it `PLATFORM=libreelec`. no idea if this is reasonable. i bet not
